server/app/controllers/userActions.js

```diff
@@ -25,19 +25,6 @@ const read = async (req, res, next) => {
   }
 };
 
-// Edit
```

Comment:
If we get rid of the `edit` function, how does a user update their own information from the `UserPage` ?

Reply:
nvm